app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -125,6 +125,8 @@ class ResponseRedacter
 
         # @param [Hash] body
```

Comment:
Should we update the parameter type if we're expecting `nil` or other non-hash values?

Reply:
I'm only expecting those values in error situations; they are not the way that the method is intended to be used. So, I would say no, non-hash values should not be documented because they're not supposed to be used. But, I could see the argument the other way, too.

Reply:
I think an array would be a not-expected value, but I do think `nil` is sort of expected, so I'd at least document that

Reply:
I think if we're handling those other value types in this method, they are supported types for the method. And if we don't want them to be supported types, we shouldn't call the method with those values.